### PR TITLE
Update mm-whats-new.md

### PR DESCRIPTION
**Pull request by yaratnaparkhi**
Added coming soon section.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/mm-whats-new.md [hunk 1]
```diff
@@ -23,7 +23,13 @@ description: "Learn about the new features and updates to existing features in M
 We're continuously adding new features to Migration Manager and fixing issues we learn about. Here's a summary of what's included.   
 
 You can help us improve Migration Manager by sending us your suggestions and reporting bugs you encounter. At the bottom of this page, click the Feedback button.
-  
+
+## Coming soon
+
+- **Support replacing invalid characters**. To help migrate files with invalid characters in the file name, users will be able to specify pre-defined characters to replace invalid characters. 
+- **Select a Teams as a destination**. Users will be able to select a Teams team and the channel as the migration destination.
```
Comment (JohanFreelancer9) [inline migration/mm-whats-new.md:30]:
```suggestion
- **Select a Teams team as a destination**. Users will be able to select a Teams team and the channel as the migration destination.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**